testutil/singlevm/setup.sh

```diff
@@ -327,3 +355,6 @@ fi
 #Also kill the CNCI (as there is no other way to delete it today)
 sudo killall qemu-system-x86_64
 sudo rm -rf /var/lib/ciao/instances
+sudo ip link del eth10
+sudo killall dnsmasq
```

Comment:
@obedmr can we only kill the one we started.. This should have been done by me earlier. However with your new change, running on the user's system becomes very simple (vs running in a VM) and having this change will help make it more robust.